src/main/java/com/fasterxml/jackson/datatype/guava/GuavaModule.java

```diff
@@ -3,7 +3,11 @@
 import com.fasterxml.jackson.core.Version;
 
 import com.fasterxml.jackson.databind.*;
+import com.fasterxml.jackson.databind.cfg.PackageVersion;
```

Comment:
Ok this caused the bug -- PackageVersion must not come from version of another module, like `jackson-databind`.
It is generated by Maven build. I removed this which fixes the unit test fail.